src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/SyntaxNodeExtensions.cs

```diff
@@ -303,11 +302,7 @@ public static TRoot InsertNodesAfter<TRoot>(this TRoot root, SyntaxNode nodeInLi
 
     public static string GetContent<TNode>(this TNode node) where TNode : SyntaxNode
     {
-        using var _ = StringBuilderPool.GetPooledObject(out var builder);
-        using var writer = new System.IO.StringWriter(builder);
```

Comment:
>StringWriter(builder) [](http://example.com/codeflow?start=41&length=21)

Here's where GetContent implicity specified CultureInfo.CurrentCulture

Reply:
I really don't think you need to worry about `CultureInfo` for the `StringWriter`. `GreenNode.WriteTo` delegates to `GreenNode.WriteTokenTo` to do any actually writing. `WriteTokenTo` is only overridden by `SyntaxToken` and it just writes a string. IOW, nothing actually writes anything other than strings, so it's essentially culture-neutral.

Reply:
That's what I was hoping. Went ahead and changed to always use InvariantCulture. Thanks!